packages/ia-components/sandbox/track-lists/track-utils.js

```diff
@@ -0,0 +1,67 @@
+/**
+ * For a given track in an album, it formats the track title value
+ * @param { string } trackTitle
+ * @param { string } fileName
+ */
+const formatTrackTitle = (trackTitle = '', fileName = '') => {
+  if (trackTitle) {
+    return trackTitle;
+  }
+  // strip file extension from its name and return
+  return fileName.replace(/.mp3/, '');
+};
+
+/**
+ * For a given track in an album, it formats the track artist value
+ * This follows the structured metadata for collections in: `acdc`, `album_recordings`, & `78rpm`
+ *
+ * @param { string } trackArtist - delimited by `;\s|;`;
+ * @param { array|string } albumCreator
+ * @param { string } albumTitle
+ *
+ * @returns { string }
+ */
+const formatTrackArtist = (trackArtist = '', albumCreator = '', albumTitle = '') => {
+  const titleHasTrackArtist = albumTitle.includes(trackArtist);
+
+  // if album title has artist name => no
+  // we are assuming that this is a "Best of <artist>" album
+  if (titleHasTrackArtist) {
+    return '';
+  }
+
+  if (trackArtist === albumCreator) {
+    return '';
+  }
+
+  // Track Artists come in a delimited string
+  // `;\s` => acdc, album_recordings
+  // `;` => 78rpm
+  const parseArtistDelimiter = new RegExp(/;\s|;/g);
```

Comment:
This regex could be `/;\s?/g` and will perform the same checks, giving you an optional space character rather than an either|or